views/services.hbs

```diff
@@ -0,0 +1,28 @@
+<section class="">
+  <div class="">
+    <h2>{{text.serviceQuesion}}</h2>
+  </div>
+  <div class="">
+    <a href="home" aria-label={{text.back}}><img src="./assets/icons/BACK_btn.svg" ></a>
+  </div>
+  <div id="services-buttons" class="">
+    <button type="button" aria-pressed="false">{{text.food}}</button>
+    <button type="button" aria-pressed="false">{{text.shopping}}</button>
+    <button type="button" aria-pressed="false">{{text.religion}}</button>
+    <button type="button" aria-pressed="false">{{text.health}}</button>
+    <button type="button" aria-pressed="false">{{text.viewPoints}}</button>
+    <button type="button" aria-pressed="false">{{text.nature}}</button>
+    <button type="button" aria-pressed="false">{{text.museums}}</button>
+  </div>
+  <section>
+    <p>Filters chosen</p>
```

Comment:
"Filters Chosen" should be a key in the text object so we can arabic and english